css/reset.css

```diff
@@ -1 +1,50 @@
 /* Base styles */
+
+:root{
+    --headercolor: #3C3A39;
+    --maincolor: #1C1A19;
+}
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body {
+    font-weight: normal;
+    background-color: white;
+    color: var(--maincolor);
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+}
+.group::after
+{
+clear: both;
+content: ".";
+display: block;
+height: 0px;
+line-height: 0px;
+visibility: hidden;
+}
+h1{
```

Comment:
This is not well-formatted. Maybe we somehow got to bypass it but this should be formatted well

Reply:
This is still not well-formatted. Just confused as to why your stylelint is not catching this.